skops/_min_dependencies.py

```diff
@@ -11,7 +11,8 @@
 #     "tomli": ("1.1.0", "install", "python_full_version < '3.11.0a7'"),
 dependent_packages = {
     "scikit-learn": ("0.24", "install", None),
-    "huggingface_hub": ("0.9.0rc3,<0.10.0rc0", "install", None),
+    # TODO: remove '<0.12.0rc0' once we changed hub calls to use use_auth_token
+    "huggingface_hub": ("0.9.0rc3,<0.12.0rc0", "install", None),
```

Comment:
```suggestion
    "huggingface_hub": ("0.9.0rc3,<0.11.0rc0", "install", None),
```

We don't know what will break in the next release.